lib/natural/distance/jaro-winkler_distance.js

```diff
@@ -26,11 +26,14 @@ THE SOFTWARE.
 // http://en.wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance
 // s1 is the first string to compare
 // s2 is the second string to compare
-function distance(s1, s2) {
+// caseSensitive is a boolean expression determining whether 'a' matches 'A'
+function distance(s1, s2, caseSensitive) {

```

Comment:
You should fix this so that any old code still works as expected, i.e. the third flag should be `ignoreCase` so that when it's true you can add new functionality